### ISSUE TITLE
Toggle comment doesn't work

### ISSUE THREAD

**Issue by mkraska** [fiziko/language-cgx#3]
Edit > Toggle Comment 
just inserts block comment marks (which are not valid in CGX) instead of the # line comment mark
This used to work in earlier versions but is broken in 0.15

Also, if possible, could the key-binding for toggle comment be changed from Ctrl-/ to something else which can be typed in a german keyboard?

**Comment by fiziko** [comment]
Should be fixed in 1.6.0 - let me know.

Regarding keybindings: This is not related to language-cgx, since language-cgx does not define keybindings.

To change the keybinding, read this: https://atom.io/docs/v1.4.0/behind-atom-keymaps-in-depth

OR:
1) Go to Settings-View -- e.g. using keybinding ctrl-,
2) Enter "comment" in the search field and check the current keybinding
3) Open keymap-file:
![screenshot](https://cloud.githubusercontent.com/assets/5688765/12372511/cd68b4f8-bc5a-11e5-82c4-661996c6d98c.png)
4) Redefine, e.g, set F12 to toggle comment

```javascript
'atom-workspace atom-text-editor:not([mini])':
  'f12': 'editor:toggle-line-comments'
```

**Comment by mkraska** [comment]
Thanks a lot, works now!
